modules/geographiclib/2.4.0.bcr.1/MODULE.bazel

```diff
@@ -0,0 +1,15 @@
+module(
+    name = "geographiclib",
+    version = "2.4.0.bcr.1",
+    bazel_compatibility = [">=7.2.1"],
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "cmake_configure_file", version = "0.1.0")
+bazel_dep(name = "rules_cc", version = "0.1.0")
+bazel_dep(name = "rules_python", version = "1.0.0")
+python = use_extension("@rules_python//python/extensions:python.bzl", "python")
+python.toolchain(
+    is_default = True,
```

Comment:
Do tests pass without this line? That would seem safer based on https://rules-python.readthedocs.io/en/latest/toolchains.html.